isso/js/app/config.js

```diff
@@ -1,28 +1,14 @@
 const utils = require("app/utils");
+const default_config = require("app/default_config");
 
 "use strict";
 
-var config = {
-    "css": true,
-    "css-url": null,
-    "lang": "",
-    "default-lang": "en",
-    "reply-to-self": false,
-    "require-email": false,
-    "require-author": false,
-    "reply-notifications": false,
-    "max-comments-top": "inf",
-    "max-comments-nested": 5,
-    "reveal-on-click": 5,
-    "gravatar": false,
-    "avatar": true,
-    "avatar-bg": "#f0f0f0",
-    "avatar-fg": ["#9abf88", "#5698c4", "#e279a3", "#9163b6",
-                  "#be5168", "#f19670", "#e4bf80", "#447c69"].join(" "),
-    "vote": true,
-    "vote-levels": null,
-    "feed": false
-};
+// Preserve default values to filter out when comparing
+// with values fetched from server
+var config = {};
+for (let key in default_config) {
```

Comment:
If anyone has a cleaner way to clone a dict-like objekt in JS, please let me know.

Reply:
What about this?
```js
var config = { ...default_config };
```

Reply:
Destructuring was on my "not-sure-we-can-do-that-without-throwing-old-browsers-under-the-bus"-list. But I've used that syntax for the template files already, so I might use it here as well.

Btw, I really appreciate the two of you, @Kerumen and @adroste, weighing in on these issues. It's a lot of work to wrangle all this JS and nice to have good advice from people more knowledgeable on this topic.